compiler/record-hessian/include/record-hessian/HessianVector.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __RECORD_HESSIAN_HESSIANVECTOR_H__
+#define __RECORD_HESSIAN_HESSIANVECTOR_H__
+
+#include <vector>
+#include <cstddef>
+#include <unordered_map>
+#include <luci/IR/CircleNodes.h>
+
+namespace record_hessian
+{
+
+using HessianMap = std::unordered_map<const luci::CircleNode *, std::vector<float>>;
```

Comment:
I think this has to be moved to `HessianComputer.h`? `HessianMap` is not used anywhere in this file.